clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -153,8 +153,9 @@ public void unsubscribe() {
         for (Map.Entry<TopicPartition, List<ConsumerRecord<K, V>>> entry : this.records.entrySet()) {
             if (!subscriptions.isPaused(entry.getKey())) {
                 List<ConsumerRecord<K, V>> recs = entry.getValue();
-                if (!recs.isEmpty())
+                if (!recs.isEmpty()) {
```

Comment:
Is the Kafka coding convention to do braces around single-line if statements? Because I've noticed them being skipped before, but I didn't speak up because I didn't know what your conventions are.